analyzers/tests/SonarAnalyzer.Test/TestCases/NamedPlaceholdersShouldBeUnique.cs

```diff
@@ -45,10 +45,17 @@ void Compliant(ILogger logger, string foo, string bar)
             logger.LogInformation("Hey {{foo}} and {{foo}}", foo, foo);      // Compliant
             logger.LogInformation("Hey {foo} and {{foo}}", foo, foo);        // Compliant
 
-            // Not string literal
+            // Contains Interpolation
             logger.LogWarning($"Hey {foo} and {foo}", foo, foo);             // Compliant
+            logger.LogWarning($"Hey {foo}" + "and {foo}", foo, foo);         // Compliant
+            logger.LogInformation("Hey {foo}" + $"and {foo}" + "{foo}", foo, foo); // Compliant FN
```

Comment:
We completely exclude cases where they contain `Interpolation`. 

`InterpolatedText` is fine:

![image](https://github.com/SonarSource/sonar-dotnet/assets/115458417/464bc3e6-7826-40f5-b86c-a6120408e42a)

`AddExpression` is fine:

![image](https://github.com/SonarSource/sonar-dotnet/assets/115458417/40a81118-ff69-4fda-921b-ce939690874f)

`Interpolation` is NOT fine:

![image](https://github.com/SonarSource/sonar-dotnet/assets/115458417/c90faa2b-2c9d-4547-b1ca-bc70faad48d5)